react-native-libraries.json

```
@@ -10016,5 +10016,12 @@
     ],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/anday013/react-native-otp-entry",
+    "images": ["https://github.com/anday013/react-native-otp-entry/blob/main/otp_demo.gif"],
+    "ios": true,
+    "android": true,
+    "expo": true,
```

```suggestion
    "expo": true
```

Thanks! Currently CI fails due to tailing comma.